catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/IngestCommand.java

```diff
@@ -491,41 +491,19 @@ private CreateResponse createMetacards(CatalogFacade catalog, List<Metacard> lis
   private Metacard readMetacard(File file) throws IngestException {
     Metacard result = null;
 
-    FileInputStream fis = null;
-    ObjectInputStream ois = null;
-
-    try {
+    try (FileInputStream fis = new FileInputStream(file);
+        ObjectInputStream ois = new ObjectInputStream(new FileInputStream(file)); ) {
       if (SERIALIZED_OBJECT_ID.matches(transformerId)) {
-        ois = new ObjectInputStream(new FileInputStream(file));
         result = (Metacard) ois.readObject();
```

Comment:
I am actually not sure what the intent was there. I see in bellow PR they just added that `if (StringUtils.isBlank(result.getTitle()))` check for the one case. Don't know if they purposely did not add it here as well, maybe other was able to title blank? Do you think we should add same title check here as well?
https://github.com/codice/ddf/commit/e3789e8dc2ee76bd8ba7729934f7f12e76f60e57#diff-20be4da2d423f8e4c89eb37c225627ed147857fae1f4433dbb4f2ae223e2f0efR284

Reply:
I don't know whether we should add the same check. Whether we should sort of exceeds the scope of your PR, so we can ignore the question, but I thought that I'd ask in case anyone knows why the code checks for a blank title in one case but not in the other.